.github/workflows/publish.yml

```diff
@@ -20,3 +20,29 @@ jobs:
       - run: yarn changeset publish
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+  send-telegram-message:
```

Comment:
no separate job , add its steps to publish job

Reply:
it is more clean!